tests/integration_tests.rs

```diff
@@ -264,3 +265,59 @@ fn report_missed_log_items() -> Result<(), ModalityProbeError> {
 
     Ok(())
 }
+
+proptest! {
+    #[test]
+    fn reports_never_end_between_multi_log_items(
+        num_events in 0_usize..256_usize,
+        num_events_with_payload in 0_usize..256_usize,
+        report_buffer_space in 0_usize..256_usize,
+    ) {
+        let mut storage = [0u8; 512];
+        let probe = ModalityProbe::try_initialize_at(&mut storage, 1).unwrap();
+        let event_a = EventId::new(2).unwrap();
+        let event_b = EventId::new(3).unwrap();
+        let event_b_payload = 4;
```

Comment:
I think this would be a stronger test if the payload was allowed to vary freely. This would help cover the cases where the payload might, incidentally, have its "clock bit" set or its "event-with-payload bit" set and we have to rely on the correctness of the log-management to avoid misinterpretation.